deploy/manifests/quay-operator/0.0.1/quay-operator.clusterserviceversion.yaml

```diff
@@ -55,6 +55,8 @@ spec:
         - kind: Role
         - kind: Rolebinding
         - kind: HorizontalPodAutoscaler
+        - kind: ServiceMonitor
```

Comment:
Could you add `{"kind": "monitoring", "managed": true}` to the list of components in the `alm-examples` annotation on the CSV?